src/lightning/pytorch/loggers/neptune.py

```diff
@@ -376,12 +379,12 @@ def training_step(self, batch, batch_idx):
     @rank_zero_experiment
     def run(self) -> "Run":
         if _NEPTUNE_AVAILABLE:
```

Comment:
Please use a better, more desriptive name, as it is not about Neptune but version...